pages/common/ln.md

```diff
@@ -4,16 +4,16 @@
 
 - create a symbolic link to a file or folder
 
-`ln -s {{path/to/original/file}} {{path/to/link}}`
+`ln -s {{path/to/target}} {{path/to/link}}`
 
 - overwrite a symbolic link to a file
 
-`ln -sf {{path/to/new/original/file}} {{path/to/file/link}}`
+`ln -sf {{path/to/new/target/file}} {{path/to/file/link}}`
 
 - overwrite a symbolic link to a folder
 
-`ln -sfT {{path/to/new/original/file}} {{path/to/folder/link}}`
+`ln -sfT {{path/to/new/target/folder/}} {{path/to/folder/link}}`
```

Comment:
I don't think this exists in BSD. Is it a very common usage / worth splitting the pages for?